### DISCUSSION TITLE
v0.4.4

### DISCUSSION THREAD

**Discussion by lbbrhzn** [lbbrhzn/ocpp#297] [Announcements]
## Changes

* Fall back to ocpp=0.11.0 (#294) @lbbrhzn
* Add sampling config of meter when idle (#292) @drc38
* Rebase pullrequests (#293) @lbbrhzn
* Update requirements.txt (#291) @lbbrhzn
* switch HAChargerDetails to Diagnostic category (#270) @drc38

## :package: Dependencies

* build(deps): bump release-drafter/release-drafter from 5.15.0 to 5.16.1 (#289) @dependabot
* build(deps): bump pytest-homeassistant-custom-component from 0.5.11 to 0.5.12 (#284) @dependabot
* build(deps): bump pytest-homeassistant-custom-component from 0.5.10 to 0.5.11 (#277) @dependabot
* build(deps): bump pytest-homeassistant-custom-component from 0.5.9 to 0.5.10 (#269) @dependabot
* build(deps): bump hacs/action from 21.12.0 to 21.12.1 (#268) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/lbbrhzn/ocpp/releases/tag/v0.4.4'>v0.4.4</a>.</em>

**Comment by GrantKeymer** [comment]
Thank you very much for this new release.  HA offered me 0.4.5 as the latest version, which I've installed and my EV is happily charging, with most of the measurands working for the first time.  I hope the slide control 'Charger.Maximum_Current' will be available soon.  At present I need to adjust the charging current from MyWallbox app, but it gets the job done.

Great job!

**Comment by lbbrhzn** [comment]
Sweet as, chur bro! She'll be right.
